manifests/profile/haproxy.pp

```diff
@@ -43,7 +43,7 @@
   }
 
   $services.filter |$service, $params| {
-    $params.has_key('floating_ip')
+    'floating_ip' in $params
```

Comment:
I forget whether this is something missed from a while ago that was like, no idea how this is still working—or if it's deprecated, and time to update the syntax, but either way, looks familiar. Updated stdlib and updated puppet language.

Reply:
This was merged in #682, so it'll won't be in this PR once I move it out of draft mode. Anyway, neither syntax makes a ton of sense to me, but this passes the tests w/ either version of stdlib, the old syntax only passes on the outdated stdlib, and I checked just to be sure the code isn't a noop: replacing the line w/ `true` or `false` fails CI.